### ISSUE TITLE
Where did the option's box go?

### ISSUE THREAD

**Issue by double-beep** [soscripted/sox#385]
When I clicked at the settings icon to select some options, the box appeared...:

![image](https://user-images.githubusercontent.com/38133098/54084208-8e86ba80-4336-11e9-9b98-600aa2abbacd.png)

there, at the end of my browser window, very very left. How can I fix that? Can I move it?

Using Windows 10, version 10.0.17763.348 and Microsoft Edge 44.17763.1.0.

**Comment by shu8** [comment]
Thanks for the bug report @double-beep. I haven't tried this script in Edge so this may be a bug on Edge. What userscript manager are you running?

I'll try running it on Edge as soon as I can and see if I can fix it :)

Just wanted to clarify the issue -- the settings dialog appears right at the edge of the screen and not under the settings button like it should?

**Comment by double-beep** [comment]
@shu8 first, I should clarify I use Tampermonkey v4.8.5847 as a userscript manager.

Hmmm... I think you misunderstood my actual issue. The thing that  the settings dialog appears right at the edge of the screen and not under the settings button like it should is a part of the issue. It appears:

![image](https://user-images.githubusercontent.com/38133098/54198049-2834a000-44ce-11e9-9e7a-79fa9e72e74d.png)

note that the black part is **my desktop**. Even if I maximize the window, I cannot see it. I see it a little bit bigger, but not the whole.

**Comment by shu8** [comment]
Thanks for clarifying, it seems quite weird :/

I can't reproduce on Edge with Tampermonkey. Could you try using the [latest dev version](https://github.com/soscripted/sox/raw/dev/sox.user.js) and see if it's fixed in there?

If not, do you have any other userscripts or extensions running that might be conflicting with SOX? Please could you try disabling any if you do and see if it works then?

**Comment by double-beep** [comment]
@shu8 it worked after I disabled all userscripts except this one. I will take a look to the one that conflicts with it and see if I can fix it.

Thanks you very much!

**Comment by shu8** [comment]
Great! If you figure out the extension do let me know and I'll see if there's an easy fix I can do (no promises though! 😛 ) :)

**Comment by double-beep** [comment]
Yeah, I found the con

**Comment by double-beep** [comment]
@shu8 I found the userscript which conflicts; it is [this one](https://github.com/samliew/SO-mod-userscripts/blob/master/PostIdsEverywhere.user.js). Take a look at it, if you can't do anything, then just leave it as is.

**Comment by shu8** [comment]
@double-beep Thanks, I found the CSS rule that's causing it (https://github.com/samliew/SO-mod-userscripts/blob/master/PostIdsEverywhere.user.js#L80) but I'm not entirely sure how to fix it. I tried messing around the various CSS `right` and `left` positions but couldn't get it to work, sorry :/

If that script works for you if you remove the line I linked on your own version, then that should make it work for you again. 
If not, I can consider making the dialog draggable, although I would have to look into that more if needed and think about what it might look like!
